logging/monolog_console.rst

```diff
@@ -38,19 +38,19 @@ The example above could then be rewritten as::
     namespace App\Command;
 
     use Psr\Log\LoggerInterface;
+    use Symfony\Component\Console\Attribute\AsCommand;
     use Symfony\Component\Console\Command\Command;
-    use Symfony\Component\Console\Input\InputInterface;
-    use Symfony\Component\Console\Output\OutputInterface;
+    use Symfony\Component\Console\Style\SymfonyStyle;
 
-    class YourCommand extends Command
+    #[AsCommand(name: 'app:my-command')]
+    class MyCommand
```

Comment:
`// src/Command/YourCommand.php` L37 must also be renamed to `// src/Command/MyCommand.php`